src/Loader/PoLoader.php

```diff
@@ -26,9 +26,14 @@ public function loadString(string $string, Translations $translations = null): T
             //Multiline
             while (substr($line, -1, 1) === '"'
                 && $nextLine !== false
-                && substr(trim($nextLine), 0, 1) === '"'
+                && (substr(trim($nextLine), 0, 1) === '"' || substr(trim($nextLine), 0, 4) === '#~ "')
             ) {
-                $line = substr($line, 0, -1).substr(trim($nextLine), 1);
+                if (substr(trim($nextLine), 0, 1) === '"') { // Normal multiline
+                    $line = substr($line, 0, -1).substr(trim($nextLine), 1);
+                }
+                if (substr(trim($nextLine), 0, 4) === '#~ "') { // Disabled multiline
```

Comment:
Here I think it's better a `else if`, so if the first condition is true, this second is not evaluated